scripts/verify-typescript.js

```diff
@@ -4,25 +4,41 @@
 
 // This file converts example input into selector utils and checks that the result compiles by Typescript
 const fs = require('fs');
+const glob = require('glob');
 const path = require('path');
 const { execSync } = require('child_process');
 const convertToSelectorUtil = require(path.join(__dirname, '../lib/converter/dist')).default;
 
+// These test files include utils represented as external dependencies to properly test their conversion to selectors.
+// We need to skip them here because they are not expected to compile correctly.
+const skip = ['strip-external-imports.ts'];
```

Comment:
This change and everything else in this file seems like too much work to work around this issue.

Alternative options

1. Can we make that code compile using `paths` or `typeRoots` options from tsconfig.
2. If fixing the typecheck is hard, can we put them into a separate `converter-no-typecheck` folder (seems easier than writing that much code to skip, right?)